clusters/build-clusters/01_cluster/ci-search/deploy.yaml

```diff
@@ -67,6 +67,9 @@ spec:
           requests:
             cpu: 100m
             memory: 1Gi
+          limits:
+            cpu: 200m
```

Comment:
Both this and the CPU request seem highly optimistic for what the service does.

Do we have an idea of the average utilization?

Reply:
The CPU limits should be fine, but the mem limit is debatable. Sometimes it goes to 20Gi